src/pages/components/index.html

```diff
@@ -3,13 +3,14 @@
         <h1 class="title">Components</h1>
     </div>
 
-    <div class="row text input">
+    <div class="row text input" id="text-input" >
         <div class="container-text-input-box">
             <h3>Inputs</h3>
 
             <p>Normal</p>
 
-            <input class="input-box" type="text" placeholder="First name" />
+            <input class="input-box" id="Text-input-box" type="text" placeholder="First name"  name="text"/>
+            <input class="input-box" id="My-Email" name="email" type="email">
```

Comment:
Where Placeholder??? Organize também os componentes, coloque o título do tipo de componente assim como os demais. Email input is not a normal input 
![image](https://user-images.githubusercontent.com/23269934/162940013-d5d3066a-abbf-459a-b607-025fba15cc6e.png)

Reply:
fixed